### PR TITLE
init commit

### PR DESCRIPTION
**Pull request by Elyas-Amini**

### CODE REVIEW

File: .github/workflows/dotnet-linux.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        with:
+          dotnet-version:  |
+            3.1.x
+            5.0.x
+            6.0.x
+            7.0.x
+      - name: Install Android
+        run: dotnet workload install android
+      - name: Restore dependencies
+        run: dotnet restore ./src/CSharp/EasyMicroservices.WhiteLabelsMicroservice.sln
```
Comment (Ali-YousefiTelori) [inline .github/workflows/dotnet-linux.yml]:
```suggestion
        run: dotnet restore ./src/CSharp/EasyMicroservices.CommentsMicroservice.sln
```

File: .github/workflows/dotnet-linux.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      - name: Build
+        run: dotnet build ./src/CSharp/EasyMicroservices.WhiteLabelsMicroservice.sln --no-restore
```
Comment (Ali-YousefiTelori) [inline .github/workflows/dotnet-linux.yml]:
```suggestion
        run: dotnet build ./src/CSharp/EasyMicroservices.CommentsMicroservice.sln --no-restore
```

File: .github/workflows/dotnet-linux.yml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+      - name: Test
+        run:  |
+          dotnet test ./src/CSharp/EasyMicroservices.WhiteLabelsMicroservice.sln --no-build --verbosity normal -f netcoreapp3.1
```
Comment (Ali-YousefiTelori) [inline .github/workflows/dotnet-linux.yml]:
```suggestion
          dotnet test ./src/CSharp/EasyMicroservices.CommentsMicroservice.sln --no-build --verbosity normal -f netcoreapp3.1
```

File: .github/workflows/dotnet-linux.yml [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+          dotnet test ./src/CSharp/EasyMicroservices.WhiteLabelsMicroservice.sln --no-build --verbosity normal -f net6.0
```
Comment (Ali-YousefiTelori) [inline .github/workflows/dotnet-linux.yml]:
```suggestion
          dotnet test ./src/CSharp/EasyMicroservices.CommentsMicroservice.sln --no-build --verbosity normal -f net6.0
```

File: .github/workflows/dotnet-linux.yml [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+          dotnet test ./src/CSharp/EasyMicroservices.WhiteLabelsMicroservice.sln --no-build --verbosity normal -f net5.0
```
Comment (Ali-YousefiTelori) [inline .github/workflows/dotnet-linux.yml]:
```suggestion
          dotnet test ./src/CSharp/EasyMicroservices.CommentsMicroservice.sln --no-build --verbosity normal -f net5.0
```

File: .github/workflows/dotnet-macos-client.yml [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+        with:
+          dotnet-version:  |
+            3.1.x
+            5.0.x
+            6.0.x
+            7.0.x
+      - name: Install Android
+        run: dotnet workload install android
+      - name: Restore dependencies
+        run: dotnet restore ./src/CSharp/EasyMicroservices.WhiteLabelsMicroservice.Client.sln
```
Comment (Ali-YousefiTelori) [inline .github/workflows/dotnet-macos-client.yml]:
```suggestion
        run: dotnet restore ./src/CSharp/EasyMicroservices.CommentsMicroservice.Client.sln
```

File: .github/workflows/dotnet-macos-client.yml [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+      - name: Build
+        run: dotnet build ./src/CSharp/EasyMicroservices.WhiteLabelsMicroservice.Client.sln --no-restore
```
Comment (Ali-YousefiTelori) [inline .github/workflows/dotnet-macos-client.yml]:
```suggestion
        run: dotnet build ./src/CSharp/EasyMicroservices.CommentsMicroservice.Client.sln --no-restore
```

File: .github/workflows/dotnet-macos-client.yml [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+      - name: Test
+        run:  |
+          dotnet test ./src/CSharp/EasyMicroservices.WhiteLabelsMicroservice.Client.sln --no-build --verbosity normal -f netcoreapp3.1
```
Comment (Ali-YousefiTelori) [inline .github/workflows/dotnet-macos-client.yml]:
```suggestion
          dotnet test ./src/CSharp/EasyMicroservices.CommentsMicroservice.Client.sln --no-build --verbosity normal -f netcoreapp3.1
```

File: .github/workflows/dotnet-macos-client.yml [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+          dotnet test ./src/CSharp/EasyMicroservices.WhiteLabelsMicroservice.Client.sln --no-build --verbosity normal -f net6.0
```
Comment (Ali-YousefiTelori) [inline .github/workflows/dotnet-macos-client.yml]:
```suggestion
          dotnet test ./src/CSharp/EasyMicroservices.CommentsMicroservice.Client.sln --no-build --verbosity normal -f net6.0
```

File: .github/workflows/dotnet-macos-client.yml [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+          dotnet test ./src/CSharp/EasyMicroservices.WhiteLabelsMicroservice.Client.sln --no-build --verbosity normal -f net5.0
```
Comment (Ali-YousefiTelori) [inline .github/workflows/dotnet-macos-client.yml]:
```suggestion
          dotnet test ./src/CSharp/EasyMicroservices.CommentsMicroservice.Client.sln --no-build --verbosity normal -f net5.0
```

File: README.md [hunk 11]
```diff
@@ -1 +1 @@
-# Comments-Microservice
+# Tenants-Microservice
```
Comment (Ali-YousefiTelori) [inline README.md]:
```suggestion
# Comments-Microservice
```

**Review by Ali-YousefiTelori — CHANGES_REQUESTED** [review]

**Review by mahdidelzendeh72 — APPROVED** [review]

**Review by mahdidelzendeh72 — APPROVED** [review]

File: src/CSharp/EasyMicroservices.CommentsMicroservice.Database/Database/Schemas/CommentSchema.cs [hunk 12]
```diff
@@ -0,0 +1,18 @@
...
+
+namespace EasyMicroservices.CommentMicroservice.Database.Schemas
+{
+    public class CommentSchema
+    {
+        public string Name { get; set; }
+        public string Text { get; set; }
+        public string Email { get; set; }
+        public string Website { get; set; }
+        public DateTime CreationTime { get; set; }
```
Comment (Ali-YousefiTelori) [inline src/CSharp/EasyMicroservices.CommentsMicroservice.Database/Database/Schemas/CommentSchema.cs]: Change it to CreationDateTime

File: src/CSharp/EasyMicroservices.CommentsMicroservice.Database/Database/Schemas/CommentSchema.cs [hunk 13]
```diff
@@ -0,0 +1,18 @@
...
+        public DateTime ModifiationTime { get; set; }
```
Comment (Ali-YousefiTelori) [inline src/CSharp/EasyMicroservices.CommentsMicroservice.Database/Database/Schemas/CommentSchema.cs]: Change it to ModifiationDateTime

File: src/CSharp/EasyMicroservices.CommentsMicroservice.Domain/Contracts/Common/CommentContract.cs [hunk 14]
```diff
@@ -0,0 +1,19 @@
...
+namespace EasyMicroservices.CommentsMicroservice.Contracts.Common
+{
+    public class CommentContract
+    {
+        public long Id { get; set; }
+        public string Name { get; set; }
+        public string Text { get; set; }
+        public string Email { get; set; }
+        public string Website { get; set; }
+        public DateTime CreationTime { get; set; }
```
Comment (Ali-YousefiTelori) [inline src/CSharp/EasyMicroservices.CommentsMicroservice.Domain/Contracts/Common/CommentContract.cs]: Rename creation and modification same as entity

**Review by Ali-YousefiTelori — CHANGES_REQUESTED** [review]

**Review by Ali-YousefiTelori — APPROVED** [review]
Thank you @ElyasAmini01 for your contribution.

**Pull request merged**